src/OnchainKitConfig.ts

```diff
@@ -9,11 +9,7 @@ export const ONCHAIN_KIT_CONFIG: OnchainKitConfig = {
   chain: baseSepolia,
   rpcUrl: null,
   schemaId: null,
-  walletCapabilities: {
-    hasAtomicBatch: false,
-    hasAuxiliaryFunds: false,
-    hasPaymasterService: false,
-  },
+  walletCapabilities: {},
```

Comment:
We should not expose this at all here, but instead being part of his own Hook.

Also this change will trigger a braking change, so we need to time this merge correctly.